lib/options.js

```diff
@@ -86,13 +87,14 @@ module.exports = optionator({
         option: "ignore",
         type: "Boolean",
         default: "true",
-        description: "Use .eslintignore"
+        description: "Disable use of .eslintignore"
     },
     {
         option: "color",
+        alias: "colour",
         type: "Boolean",
```

Comment:
This is incorrect, the default is false. `--color` is handled directly by node, we just pass it through.